conf/defaults.ini

```diff
@@ -62,6 +62,13 @@ password =
 # For "postgres" only, either "disable", "require" or "verify-full"
 ssl_mode = disable
 
+# For "mysql" only, either true, false, skip-verify
+tls_mode = false
```

Comment:
The dsn structure for having ssl is different for postgres and mysql. They accept different values